src/vstart.sh

```diff
@@ -754,10 +764,18 @@ start_osd() {
     for osd in `seq 0 $(($CEPH_NUM_OSD-1))`
     do
 	if [ "$new" -eq 1 ]; then
+	    if [ "$use_nvme_ssd" -eq 1 ]; then
+		    wconf <<EOF
+[osd.$osd]
+        host = $HOSTNAME
```

Comment:
no need to replicate this part. how about:

```sh
if [ "$new" -eq 1 ]; then
  wconf <<EOF
[osd.$osd]
    host = $HOSTNAME
EOF
  if [ "$use_nvme_ssd" -eq 1 ]; then
    wconf <<EOF
    bluestore_block_path = spdk:$(get_pci_selector $((osd+1)))
EOF
  fi
  # ...
fi
```

in which, i'd recommend replace "use_nvme_ssd" with "spdk_enabled".